integrations/clickup/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
+
+# Port_Ocean 0.0.1 (2024-07-04)
+
+### Features
+
+- New Integration for Clickup (0.0.1)
```

Comment:
change log doesn't match version in pyproject.toml

Reply:
This has been corrected, thank you for the comment